package.json

```diff
@@ -37,9 +38,9 @@
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.5.1",
-    "ember-resolver": "^2.0.3",
     "ember-lodash": "0.0.5",
-    "ember-suave": "2.0.1",
+    "ember-resolver": "^2.0.3",
+    "ember-suave": "4.0.1",
```

Comment:
@alexdiliberto Okay, cool. Yeah, we can discuss further to find the sweet spot between ember suave and linting.